docs/gallery/interactive/pyfar_filter_types.ipynb

```diff
@@ -0,0 +1,86 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview of pyfar Filter types\n",
+    "\n",
+    "The [`pyfar.dsp.filter`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.filter.html) module contains different filter types that are briefly introduced in the following. The filters can be used to directly filter pyfar *Signals* or can return a pyfar *Filter* object. For more information on this refer to the `LINK-TO-FILTERING-EXAMPLES` and `LINK-TO-AUDIO-OBJECT-EXAMPLES`."
```

Comment:
These type of links are not rendered correctly in the html version. Solutions are to remove the link or remove the ` signs between [...] :)

Reply:
Yes, thanks - done that everywhere in the notebook.